### PR TITLE
1864 binarize mojo

### PR DESCRIPTION
**Pull request by levBagryansky**
#1864 
I introduced xsl transformations in order to copy rust code to separate section in xml.
I creatred 2 @todo for the tasks to extract the code and to compile them to libraries.
The BunarizeMojoTest checks that the content from new section converted to the rust program.

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur please review

File: eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java [hunk 1]
```diff
@@ -49,13 +70,32 @@
...
+        ),
+        Arrays.asList(
+            "/org/eolang/maven/add_rust/add_rust.xsl"
+        )
+    ).back().back();
+
+    /**
+     * Extension for compiled sources in XMIR format (XML).
+     */
+    static final String EXT = "xmir";
```
Comment (Graur) [inline eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java:97]: @levBagryansky Why this field is not private?

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java (hunk 1 above)
Comment (levBagryansky) [reply eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java:97]: this field was created for the future

**Comment by levBagryansky** [comment]
@Graur according to #1874 smth wrong with windows check

**Review by Graur — APPROVED** [review]
@levBagryansky Yeah, I see. This pull request is not the cause of the broken build

**Comment by levBagryansky** [comment]
I created issue #1875 for this

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**